### PR TITLE
Increase speed of tab animation

### LINKED ISSUE #3061: Increase speed of tab animation
On cluster page when switching between e. g. Events and RBAC the line moves a bit slow, please speed it up a bit

### PR DESCRIPTION
**Pull request by kgroschoff**
**What this PR does / why we need it**:
Increase speed of tab animation

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #3061

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by kgroschoff** [comment]
@maciaszczykm / @floreks PTAL

File: src/app/shared/components/tab-card/style.scss [hunk 1]
```diff
@@ -46,4 +46,8 @@
       margin: 0;
     }
   }
+
+  mat-ink-bar {
+    transition: 200ms cubic-bezier(0.35, 0, 0.25, 1);
```
Comment (floreks) [inline src/app/shared/components/tab-card/style.scss:51]: Can't this be merged with an already existing rule for the `.mat-ink-bar` a bit above?

Comment (kgroschoff) [reply src/app/shared/components/tab-card/style.scss]: yes you are right. Haven't seen the `.mat-ink-bar`. Done

File: src/app/shared/components/tab-card/style.scss [hunk 2]
```diff
@@ -36,6 +36,8 @@
 
     .mat-ink-bar {
       top: 0;
+      transition-duration: 200ms;
+      //transition: 200ms cubic-bezier(0.35, 0, 0.25, 1);
```
Comment (maciaszczykm) [inline src/app/shared/components/tab-card/style.scss:40]: Please remember to remove the commented part before merging.

Comment (kgroschoff) [reply src/app/shared/components/tab-card/style.scss]: damn it, happened again. sorry. fixed

Comment (maciaszczykm) [reply src/app/shared/components/tab-card/style.scss]: np 🙂

**Pull request merged**